src/interfaces/Config.ts

```diff
@@ -12,6 +12,7 @@ export interface ChainConfig {
   nativeTokenPrecision: number;
   blockExplorerName: 'Etherscan';
   blockExplorerUrl: string;
+  ccipExplorerUrl: string;
```

Comment:
I would put this in constants file because it's not related to specific chain. Url is always same.

Reply:
done